scuba/scuba.py

```diff
@@ -318,6 +319,9 @@ def get_docker_cmdline(self):
         for hostpath, contpath, options in self.__get_vol_opts():
             args.append(make_vol_opt(hostpath, contpath, options))
 
+        for _, vol in self.context.volumes.items():
+            args.append(vol.get_vol_opt())
+
```

Comment:
Should the code here confirm that the configured volumes don't conflict with the volumes automatically created by scuba above?

Reply:
It's valid concern, but I think we can safely defer this.

The preconfigured volumes include `/.scuba` (for `scubainit` and the generated command script), and the absolute path of `.scuba.yml` (same inside/outside). I imagine it's fairly unlikely for someone to add a volume that matches one of those paths. If they do, and we pass conflicting `--volume` options, `docker run` will complain:
```
$ docker run --rm -it -v /tmp/a/:/var/lib/foo -v /tmp/b/:/var/lib/foo debian:10
docker: Error response from daemon: Duplicate mount point: /var/lib/foo.
See 'docker run --help'.
```

I would like to make the internal volumes use the same mechanics, but that's going to take a bit more work (due to the order that things are set up, particularly `self.vol_opts`).